stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/logic/channel/internal/ChannelLogic.kt

```diff
@@ -107,7 +107,11 @@ internal class ChannelLogic(
         if (request.isNotificationUpdate) return
         channelStateLogic.refreshMuteState()
 
-        runChannelQueryOffline(request)
+        /* It is not possible to guarantee that the next page of newer messages is the same of backend,
```

Comment:
I'm not sure where this is coming from, BLAME says it was created by the Bot in June last year.